packages/cadl-ranch-specs/http/type/union/mockapi.ts

```diff
@@ -30,3 +30,51 @@ Scenarios.Type_Union_sendSecondNamedUnionValue = passOnSuccess(
     return { status: 200 };
   }),
 );
+
+Scenarios.Type_Union_receiveString = passOnSuccess(
+  mockapi.post("/type/union/receive/string", (req) => {
+    req.expect.bodyEquals({});
```

Comment:
I don't think you need to "expect body"

Reply:
should I write `req.expect.bodyEquals(undefined)`?
Now I removed this